modules/fts/pages/fts-analyzers-search-functions.adoc

```diff
@@ -0,0 +1,33 @@
+= Analyzers - Search Functions
+
+Search functions allow users to execute full text search requests within a N1QL query. In the context of N1QL queries, a full text search index can be described as one of the following :
+* a) xref:https://docs.couchbase.com/server/current/n1ql/n1ql-language-reference/covering-indexes.html[Covering index]
+* b) Non-covering index
```

Comment:
Is there a link for non-covering indexes that we should reference like Covering Indexes?

Reply:
Couldn't find any direct links in our documentation.